.github/workflows/references.yml

```diff
@@ -0,0 +1,32 @@
+name: Generate References
+on:
+  push:
+    branches:
+      - main
+jobs:
+  run-script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
```

Comment:
```suggestion
          python-version: "3.10"
```

Reply:
I see this is marked as resolved, though I'm not seeing the suggestion being accepted or any comments. If you want to keep it with the original settings, lmk. I didn't verify that it works with the changes I proposed.

Reply:
Your right its not resolved. No, 3.10 is OK.

Reply:
See commit fe99baa